docker-compose.it.yml

```diff
@@ -8,6 +8,11 @@ services:
       - "POSTGRES_DATABASENAME=assetmgmt"
```

Comment:
reviewers note:

it appears this docker-compose file is used for integration testing https://github.com/asecurityteam/asset-inventory-api/blob/69f69be36474dc3e9415d57859c8f9a45b2c18cf/Makefile#L25

Reply:
Yes, it is